src/index.html

```diff
@@ -7,6 +7,620 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__wrapper">
+        <div class="header__logo">
+          <a href="#home" class="header__logo-link">
+            <img alt="Logo"
+            class="header__logo-img"
+            src="./images/logo.png">
```

Comment:
You need to check and fix it everywhere
<img width="1071" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/64750958/5bdad3f9-5444-435f-bba9-73a46b48aae5">